Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/service/impl/ConceptMergeService.java

```diff
@@ -0,0 +1,207 @@
+package edu.asu.conceptpower.app.service.impl;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.conceptpower.app.bean.ConceptsMergeBean;
+import edu.asu.conceptpower.app.core.ConceptTypesService;
+import edu.asu.conceptpower.app.core.ConceptTypesService.ConceptTypes;
+import edu.asu.conceptpower.app.core.IConceptManager;
+import edu.asu.conceptpower.app.error.CPError;
+import edu.asu.conceptpower.app.exceptions.DictionaryDoesNotExistException;
+import edu.asu.conceptpower.app.exceptions.DictionaryModifyException;
+import edu.asu.conceptpower.app.exceptions.IndexerRunningException;
+import edu.asu.conceptpower.app.exceptions.LuceneException;
+import edu.asu.conceptpower.app.service.IConceptMergeService;
+import edu.asu.conceptpower.core.ConceptEntry;
+
+@Service
+public class ConceptMergeService implements IConceptMergeService {
+
+    @Autowired
+    private IConceptManager conceptManager;
+
+    @Autowired
+    private ConceptTypesService conceptTypesService;
+
+    @Override
+    public ConceptsMergeBean prepareMergeConcepts(List<ConceptEntry> conceptEntries,
+            ConceptsMergeBean conceptsMergeBean) {
+        boolean typeWarning = false;
+        boolean posWarning = false;
+        conceptsMergeBean.clear();
+        for (ConceptEntry entry : conceptEntries) {
+            if (entry.getWord() != null && !entry.getWord().isEmpty()) {
+                conceptsMergeBean.setWord(conceptsMergeBean.getWord() == null
+                        || conceptsMergeBean.getWord().length() < entry.getWord().length() ? entry.getWord()
+                                : conceptsMergeBean.getWord());
+                
+                String conceptIdWithName = entry.getWord().trim() + " -- " + entry.getId();
+                conceptsMergeBean.getConceptNamesWithIds().add(conceptIdWithName);
```

Comment:
yes i do have. But the the details are split.  ConceptMergeBean contains the conceptIds and wordnetids seperately. Also the word details are not stored in the conceptMergeBean. I will create a map that has the key as ConceptId (or) wordnetid and value as the word. I will iterate the map in the UI and show what concepts are getting merged